react/components/Slider/README.md

```diff
@@ -28,3 +28,84 @@ Range slider
   range
 />
 ```
+
+Slider with programmatically controlled values
+
+```js
+const Input = require('../Input').default
+const Button = require('../Button').default
+
+const [inputValues, setInputValues] = React.useState({ left: '0', right: '99' })
+const [values, setValues] = React.useState({ left: 0, right: 99 })
```

Comment:
Have you thought about having `Input[type="number"]` and updating the `inputValues` state on Slider's change? This is so that we can see the inputs' values changing according to the Slider too.

Reply:
Nice suggestion. Done! 😉